test/celeritas/field/FieldPropagator.test.cc

```diff
@@ -42,8 +42,8 @@ namespace celeritas
 namespace test
 {
 //---------------------------------------------------------------------------//
-using constants::pi;
-using constants::sqrt_three;
+constexpr real_type pi{constants::pi};
```

Comment:
This might have been missed as part of the commit "Use 'typical' usage of constant in unit tests "